tools/templates/moar/Makefile.in

```diff
@@ -68,33 +68,32 @@ R_SETTING_MOAR = RESTRICTED.setting.moarvm
 @bpv(CLEANUPS)@ = \
   $(R_SETTING_MOAR) \
   @bpm(RUN_RAKUDO_SCRIPT)@ \
-  inst-rakudo-m@moar::obj@ \
-  inst-rakudo-debug-m@moar::obj@ \
-  inst-rakudo@moar::obj@ \
-  inst-rakudo-debug@moar::obj@ \
-  inst-rakudow-m@moar::obj@ \
-  inst-rakudow@moar::obj@ \
-@for_langalias( @bpm(INST_@uclangalias@)@ \
+@for_langalias(  @bpm(INST_@uclangalias@)@ \
```

Comment:
Minor nit, but there's inconsistent use of 0, 1, or 2 spaces after an opening `(`.

Reply:
I that specific spot it's on purpose. All the lines above and below have two leading spaces. If that line stays single space the line is indented one to few in the Makefile.